src/main/java/frc/robot/Drivetrain.java

```diff
@@ -81,8 +81,11 @@ public double deadband(double in) {
         return Math.abs(in) > DEADBAND_LIMIT ? in : 0.0;
     }
 
-    public void getTemp() {
-        left.getTemp("Left Motor Temperature");
-        right.getTemp("Right Motor Temperature");
+    /**
+     * Cools Falcon temperature.
+     */
+    public void coolFalconTemp() {
```

Comment:
Just a nitpick but I think `coolMotors` or something like that might be more descriptive.